src/pages/BrowseData.js

```diff
@@ -0,0 +1,277 @@
+    },
+    title: {
+      textTransform: 'uppercase',
+      fontSize: 16, fontWeight: 500
+    },
+    summary: {
+      fontSize: 40, fontWeight: 500, lineHeight: '52px',
+      width: 600, margin: '0.5rem 0 3rem'
+    },
+    details: {
+      fontSize: 16, textTransform: 'uppercase'
+    }
+  },
+  banner: {
+    container: {
+      display: 'flex', alignItems: 'center',
+      backgroundColor: '#170e43', color: 'white',
+      padding: `1.5rem ${sideMargin}`
+    },
+    textContainer: {
+      flex: 1
+    },
+    title: {
+      fontSize: 24, fontWeight: 500
+    },
+    details: {
+      fontSize: 24
+    },
+    buttonContainer: {
+      flex: 'none'
+    }
+  },
+  content: {
+    container: {
+      padding: `3rem ${sideMargin}`
+    },
+    header: {
+      ...Style.elements.sectionHeader, textTransform: 'uppercase'
+    },
+    title: {
+      fontSize: 20, color: Style.colors.secondary
+    }
+  }
+}
+
+
+const FadeBox = ({
+  padding = '1.5rem',
+  backgroundColor = Style.colors.background,
+  borderColor = Style.colors.textFaded,
+  fadePoint = '60%',
+  children
+}) => {
+  return div({
+    style: {
+      background: `linear-gradient(to bottom, white 0%, ${backgroundColor} ${fadePoint}`,
+      borderRadius: '8px 8px 0 0',
+      margin: `${padding} -${padding}`
+    }
+  }, [
+    div({
+      style: {
+        height: padding,
+        border: `1px solid ${borderColor}`,
+        borderBottom: 'none',
+        borderRadius: '8px 8px 0 0'
+      }
+    }),
+    div({
+      style: {
+        padding: `0 ${padding} ${padding}`,
+        borderWidth: 1,
+        borderStyle: 'solid',
+        borderImage: `linear-gradient(to bottom, ${borderColor}, ${backgroundColor} ${fadePoint}) 1 100%`,
+        borderTop: 'none',
+        borderBottom: 'none'
+      }
+    }, [children])
+  ])
+}
+
+
+const logoBox = ({ src, alt }) => div({
+  style: {
+    display: 'inline-flex', justifyContent: 'center', alignItems: 'center',
+    flex: 'none',
+    height: 200, width: 450,
+    border: `1px solid ${Style.colors.border}`, borderRadius: 5,
+    backgroundColor: 'white'
+  }
+}, [
+  img({
+    src, alt, height: '60%', width: 'auto'
+  })
+])
+
+
+const Participant = ({ logoBox, children }) => h(FadeBox, [
+  div({ style: { display: 'flex' } }, [
+    logoBox,
+    div({ style: { flex: 1, margin: '1rem 0 0 4rem' } }, [children])
+  ])
+])
+
+
+const ParticipantControls = ({ size, helpLink, children }) => {
+  return div({ style: { display: 'flex', alignItems: 'center' } }, [
+    div({ style: { width: 200, flex: 'none' } }, [size || '']),
+    children,
+    h(InfoBox, { style: { marginLeft: '1rem' }, position: 'bottom', size: 20 }, [
+      div(['Look for the Export to Saturn icon to export data from this provider. ']),
+      // not used yet:
+      helpLink && link({
+        href: helpLink,
+        target: '_blank'
+      }, ['Learn more.'])
+    ])
+  ])
+}
+
+
+const NIHCommonsButtons = div({ style: { width: 303 } }, [
```

Comment:
nit: With the included button stretch improvements, this could just be a flex column which would auto-size to the longest button.